pytest.cov

```diff
@@ -1,7 +1,5 @@
 [run]
 # omit the tests for test coverage monitoring
 omit=gatorgrouper/tests/*, .venv/*
```

Comment:
Should this be `gatorgrouper/tests/*.py` to match your include?

Reply:
Yes, I will fix that

Reply:
In the `pytest.cov` file, not the `.travis.yml` file.